site/docs/iceberg/trino.md

```diff
@@ -28,4 +28,13 @@ s3.aws-secret-key=<aws_secret_key>
        a. To create branches and tags, use the Nessie CLI available [here](https://projectnessie.org/nessie-latest/cli/).
        b. To switch references (branch or tag), update the catalog property `iceberg.nessie-catalog.ref`.
 
+Nessie can also be accessed via REST catalog path.
+Sample Nessie-REST configuration for Iceberg connector in `etc/catalog/iceberg.properties`
+```
+connector.name=iceberg
+iceberg.catalog.type=rest
+iceberg.rest-catalog.uri=http://.../iceberg
```

Comment:
nit: I'd use the example URI of `http://localhost:19120/iceberg` - similar to the example above... `...` might be confusing to some users, I guess.

Reply:
done